types/lib/createFormatter.d.ts

```diff
@@ -0,0 +1,2 @@
+declare function _exports(newrelic: object): function;
```

Comment:
you could export an interface here to reflect what `newrelic` is and ensure that it has all the proper properties and methods

Reply:
> it should be whatever the return type of winston.format is that we are exporting here.

Here's that type, for reference: https://github.com/winstonjs/logform/blob/master/index.d.ts#L32

I haven't really nailed down how to do type imports yet but I'll get there.

Reply:
> we don't have to (and shouldn't i don't think?) do anything beyond the public api.

Excellent, I'll remove the non-public API files. I think they were auto-generated anyway.